public/app/api/clients/provisioning/v0alpha1/index.ts

```diff
@@ -59,6 +61,10 @@ export const provisioningAPIv0alpha1 = generatedAPI.enhanceEndpoints({
             );
           }
         }
+        // Refetch dashboards and folders after deleting a provisioned repository
+        setTimeout(() => {
+          dispatch(refetchChildren({ parentUID: undefined, pageSize: PAGE_SIZE }));
+        }, 1000);
```

Comment:
Why do we need the timeout? Won't it have resolved correctly because we `await queryFulfilled` at the top?

Or is this because of a delay on the backend? If so would be good to have a comment explaining the reason :+1: (and the same for the other cases)